types/react-loadable/webpack.d.ts

```diff
@@ -15,7 +15,7 @@ export interface ReactLoadablePluginOptions {
     filename: string;
 }
 
-export class ReactLoadablePlugin extends webpack.Plugin {
+export class ReactLoadablePlugin extends webpack.LoaderOptionsPlugin {
```

Comment:
This is my first scribbling attempts at upgrading a plugin. It's almost certainly wrong.